### PR TITLE
Add units to parameters in ini file; update com5 resistance [com1; com5]

### PR DESCRIPTION
**Pull request by leon-wagner**
Addition of missing units to com1, com5 and com6 ini files + general improvements. Issue #464

### CODE REVIEW

File: avaframe/com5SnowSlide/com5SnowSlideCfg.ini [hunk 1]
```diff
@@ -56,16 +57,16 @@ snowSlide = 1
...
+# height of the obstacles [m]
 hRes = 100
-# resistance coeff
+# resistance coefficient
 cw = 1000
-# diameter of obstacles
+# diameter of obstacles [m]
 dRes = 10
-# spacing between obstacles
+# spacing between obstacles [m]
```
Comment (fso42) [inline avaframe/com5SnowSlide/com5SnowSlideCfg.ini:71]: Oh, this is a good point, this is not valid anymore! 
We collapsed these parameters to 1. Please take the values from com1DFACfg.ini and set really high values.

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/com5SnowSlide/com5SnowSlideCfg.ini (hunk 1 above)
Comment (leon-wagner) [reply avaframe/com5SnowSlide/com5SnowSlideCfg.ini:71]: Okay, should a description of the new parameter then be added here as well? Is this then just a resistance coefficient?

File: avaframe/com1DFA/com1DFACfg.ini [hunk 2]
```diff
@@ -337,17 +336,17 @@ mucoulomb = 0.155
...
 entTempRef = -10.0
 # J/kgK (ice)
 cpIce = 2050.0
 
 #++++++++++++ Resistance force parameters
-# height of the obstacles
+# height of obstacles [m]
 hRes = 10
 # combine parameters from above for resistance force to one parameter
 cRes = 0.003
```
Comment (leon-wagner) [inline avaframe/com1DFA/com1DFACfg.ini:352]: d

File: avaframe/com1DFA/com1DFACfg.ini [hunk 3]
```diff
@@ -337,17 +336,17 @@ mucoulomb = 0.155
...
-# initial temperature for entrainment in °C
+# initial temperature for entrainment [°C]
 entTempRef = -10.0
 # J/kgK (ice)
 cpIce = 2050.0
 
 #++++++++++++ Resistance force parameters
-# height of the obstacles
+# height of obstacles [m]
 hRes = 10
```
Comment (leon-wagner) [inline avaframe/com1DFA/com1DFACfg.ini:350]: test

File: avaframe/com1DFA/com1DFACfg.ini [hunk 4]
```diff
@@ -365,7 +364,7 @@ entShearResistance = 0
 entDefResistance = 0
```
Comment (leon-wagner) [inline avaframe/com1DFA/com1DFACfg.ini:364]: These need units and explanations as well, no?

File: avaframe/com1DFA/com1DFACfg.ini [hunk 5]
```diff
@@ -337,17 +336,17 @@ mucoulomb = 0.155
...
+# reference enthalpy to compute the new mu = mu_0 x exp(-enthalpy/enthRef) [J/kg]
 enthRef = 20000
-# initial temperature in °C
+# initial temperature [°C]
 TIni = -10
-# initial temperature for entrainment in °C
+# initial temperature for entrainment [°C]
 entTempRef = -10.0
 # J/kgK (ice)
 cpIce = 2050.0
```
Comment (leon-wagner) [inline avaframe/com1DFA/com1DFACfg.ini:346]: I believe this parameter is not explained in the documentation https://docs.avaframe.org/en/1.8.3/theoryCom1DFA.html#wet-snow-friction-type has this been added recently?

Comment (leon-wagner) [inline avaframe/com1DFA/com1DFACfg.ini:346]: test

Comment (leon-wagner) [inline avaframe/com1DFA/com1DFACfg.ini:346]: I believe this parameter is not explained in the documentation https://docs.avaframe.org/en/1.8.3/theoryCom1DFA.html#wet-snow-friction-type. Is this maybe a recent addition?

Comment (leon-wagner) [reply avaframe/com1DFA/com1DFACfg.ini:346]: cpIce

File: avaframe/com5SnowSlide/com5SnowSlideCfg.ini (hunk 1 above)
Comment (fso42) [reply avaframe/com5SnowSlide/com5SnowSlideCfg.ini]: Yes, copy the info from com1DFACfg.ini. And yes, it is a coefficient, so no units needed

File: avaframe/com1DFA/com1DFACfg.ini (hunk 4 above)
Comment (fso42) [reply avaframe/com1DFA/com1DFACfg.ini:364]: Mh, leave these as is, they are in the code but might get removed. Add above entShearResistance:
Used to determine speed loss via energy loss due to entrained mass

File: avaframe/com1DFA/com1DFACfg.ini (hunk 5 above)
Comment (fso42) [reply avaframe/com1DFA/com1DFACfg.ini:346]: It is mentioned in the text, I.e the specific heat capacity.
I guess a good idea would be to add the formular for the totalEnthalpy to the documentation right away. See com1DFA.py ll. 1247 for the relevant code. If you want to try, give it a go. The relevant documentation source is theoryCom1DFA.rst

File: avaframe/com5SnowSlide/com5SnowSlideCfg.ini (hunk 1 above)
Comment (fso42) [reply avaframe/com5SnowSlide/com5SnowSlideCfg.ini]: related issue #1025

**Review by fso42 — APPROVED** [review]

**Pull request merged**